src/Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests/CloudEventHandlerUnitTests.cs

```diff
@@ -108,16 +96,14 @@ public async Task Given_EventHandler_When_HandleCloudEventAsync_Then_ReturnsHand
         var handler = GivenCloudEventHandler(services);
 
         // When
-        var result = await handler.HandleCloudEventAsync(cloudEvent, CancellationToken.None);
+        await handler.HandleCloudEventAsync(cloudEvent, CancellationToken.None);
 
         // Then
         Assert.Single(SampleEventTestHandler.ReceivedEvents, e => e.Message == eventMessage);
-        
-        Assert.Equal(EventProcessingStatus.Handled, result);
     }
 
     [Fact]
-    public async Task Given_EventHandlersFromAssembly_When_HandleCloudEventAsync_Then_ReturnsHandledAndEventWasTreated()
+    public async Task Given_EventHandlersFromAssembly_When_HandleCloudEventAsync_Then_EventIsTreated()
```

Comment:
Nit: I know it was written 'treated' before, I personnally prefer reusing the same action verb as the when action
```suggestion
    public async Task Given_EventHandlersFromAssembly_When_HandleCloudEventAsync_Then_EventIsHandled()
```